src/alpine/3.20/amd64/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM alpine:3.20
```

Comment:
This should either have `amd64/alpine:3.20` or move it out of the `amd64` directory if we want it to be multi-arch-capable.

Reply:
I updated all of the existing amd64 alpine images given they are currently being built just for amd64.